helper.php

```diff
@@ -7,6 +7,8 @@
  * @author     Michael Hamann <michael@content-space.de>
  */
 
+use dokuwiki\File\PageResolver;
+
```

Comment:
If you follow my comment regarding MediaResolver below then you will need a corresponding line such as this:
`use dokuwiki\File\MediaResolver;`